src/combinatorics/simple_graph/clique.lean

```diff
@@ -41,6 +40,21 @@ abbreviation is_clique (s : set α) : Prop := s.pairwise G.adj
 
 lemma is_clique_iff : G.is_clique s ↔ s.pairwise G.adj := iff.rfl
 
+/-- A clique is a set of vertices whose induced graph is complete. -/
+lemma is_clique_iff_induce_eq : G.is_clique s ↔ G.induce s = ⊤ :=
```

Comment:
Can you turn this one around and make it simp?

Reply:
It's not clear to me yet that either of these should be the simp normal form -- it feels like this could get in the way.